runtime/doc/lua.txt

```diff
@@ -574,6 +574,26 @@ retained for the lifetime of a buffer but this is subject to change. A plugin
 should keep a reference to the parser object as long as it wants incremental
 updates.
 
+Parser files						*treesitter-parsers*
+
+Parsers are the heart of tree-sitter. They are shared objects (`.so`) that
+tree-sitter will search for in the `parsers` runtime directory.
+
+For a parser to be available for a given language, there must be a file named
+`{lang}.so` within the parser directory.
```

Comment:
Is what jamessan was saying applicable here? Can it be something besides `.so`? Don't know if it even matters.

Reply:
I don't think it matters, but we are sure that it works this way and that's the way @bfredl tested it.

Reply:
`.so` is for linux, dylib for mac and dll for windows. does the code search for the correct extension ? 
maybe rephrase to `{lang}.{ext}  where {ext} is "so" on linux, dylib on mac, `